commit/merkleroot/rmn/controller.go

```diff
@@ -400,7 +398,7 @@ func (c *controller) sendObservationRequests(
 			fixedDestLaneUpdateRequests = append(fixedDestLaneUpdateRequests, &rmnpb.FixedDestLaneUpdateRequest{
 				LaneSource: &rmnpb.LaneSource{
 					SourceChainSelector: request.LaneSource.SourceChainSelector,
-					OnrampAddress:       typconv.KeepNRightBytes(request.LaneSource.OnrampAddress, 20),
```

Comment:
Good catch. KeepNRightBytes() was just slipping off address bytes.
@winder : This was a bug which would break any chain whose address was longer than 20 bytes.

@huangzhen1997 : Could you please also remove the function KeepNRightBytes()? Or better yet, just delete the file typeconv/address.go

Reply:
Git blame suggests that this was done intentionally.
> * convert onRampAddr from 32bytes to 20bytes in rmn reqs
 
@dimkouv how can we support generic addresses here?